src/sinks/util/retries.rs

```diff
@@ -45,6 +45,7 @@ impl<L: RetryLogic> FixedRetryPolicy<L> {
         let next = Instant::now() + self.backoff;
         let delay = Delay::new(next);
 
+        trace!(message = "retrying request.", delay_ms = %self.backoff.as_millis());
```

Comment:
Any reason this is on the trace level? I'm unsure how noisy this is, but it seems like this is more appropriate on the info level.